packages/app/src/components/VisualizationTypeSelector/VisualizationTypeSelector.js

```diff
@@ -21,6 +21,13 @@ import VisualizationTypeMenuItem from './VisualizationTypeMenuItem';
 
 export const MAPS_APP_URL = 'dhis-web-maps';
 
+export const prepareCurrentAnalyticalObject = current => ({
```

Comment:
I would move this function to the api file, as this has nothing to do with the type selector component.
It could be part of `apiSaveAOInUserDataStore`.